checkST2110.js

```diff
@@ -413,6 +533,15 @@ const test_20_71_3 = sdp => {
   return errors;
 };
 
+// Test ST 2110-20 Section 7.1 - All video streams have rtpmap entry raw/90000
+const test_20_71_3 = (sdp, params) => {
+  let errors = checkStreamsRtpMap(sdp, params, 'raw', '90000', 'ST2110-20 Section 7.1');
```

Comment:
⬇️ It's a trivial thing, but since you've done a lot of nice clean-up of the code formatting, fixing typos, etc. I wonder whether we could be consistent in comments and messages whether to use "ST2110", "ST 2110", "SMPTE ST 2110" or "SMPTE2110" (and similarly for other SMPTE specs)? I think we know "ST" is a SMPTE spec (and we don't write "IETF RFC 4566"), so I'd write "ST 2110" everywhere.

Reply:
Made changes to use ST 2110 in all places.